AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessorTests.swift

```diff
@@ -642,6 +643,90 @@ class ItemListProcessorTests: AuthenticatorTestCase { // swiftlint:disable:this
         XCTAssertEqual(item.totpKey, String.base32Key)
     }
 
+    /// `didCompleteManualCapture` failure
+    func test_didCompleteManualCapture_failure() {
+        totpService.getTOTPConfigResult = .failure(TOTPServiceError.invalidKeyFormat)
+        let captureCoordinator = MockCoordinator<AuthenticatorKeyCaptureRoute, AuthenticatorKeyCaptureEvent>()
+        subject.didCompleteManualCapture(captureCoordinator.asAnyCoordinator(),
+                                         key: "1234",
+                                         name: "name",
+                                         sendToBitwarden: false)
+        var dismissAction: DismissAction?
+        if case let .dismiss(onDismiss) = captureCoordinator.routes.last {
+            dismissAction = onDismiss
+        }
+        XCTAssertNotNil(dismissAction)
+        dismissAction?.action()
+        waitFor(!coordinator.alertShown.isEmpty)
+        XCTAssertEqual(
+            coordinator.alertShown.last,
+            Alert(
+                title: Localizations.keyReadError,
+                message: nil,
+                alertActions: [
+                    AlertAction(title: Localizations.ok, style: .default),
+                ]
+            )
+        )
+        XCTAssertEqual(authItemRepository.addAuthItemAuthItems, [])
+        XCTAssertNil(subject.state.toast)
+    }
+
+    /// `didCompleteManualCapture` success with a locally saved item
+    func test_didCompleteManualCapture_localSuccess() throws {
+        let key = String.base32Key
+        let keyConfig = try XCTUnwrap(TOTPKeyModel(authenticatorKey: key))
+        totpService.getTOTPConfigResult = .success(keyConfig)
+        authItemRepository.itemListSubject.value = [ItemListSection.fixture()]
+        let captureCoordinator = MockCoordinator<AuthenticatorKeyCaptureRoute, AuthenticatorKeyCaptureEvent>()
+        subject.didCompleteManualCapture(captureCoordinator.asAnyCoordinator(),
+                                         key: key,
+                                         name: "name",
+                                         sendToBitwarden: false)
+        var dismissAction: DismissAction?
+        if case let .dismiss(onDismiss) = captureCoordinator.routes.last {
+            dismissAction = onDismiss
+        }
+        XCTAssertNotNil(dismissAction)
+        dismissAction?.action()
+        waitFor(!authItemRepository.addAuthItemAuthItems.isEmpty)
+        waitFor(subject.state.loadingState != .loading(nil))
+        guard let item = authItemRepository.addAuthItemAuthItems.first
+        else {
+            XCTFail("Unable to get authenticator item")
+            return
+        }
```

Comment:
⛏️ Although it's not a lint warning, from what I've seen we usually put the `else` in the same row as the `guard`:
```suggestion
        guard let item = authItemRepository.addAuthItemAuthItems.first else {
            XCTFail("Unable to get authenticator item")
            return
        }
```

Reply:
I 100% agree that looks wrong. I will fix it. But I think I'm going to go ahead and merge this PR since it's the base for the other 3 and put this change on one of the child branches. 👍  I think I may have changed this in later PRs.